release/models/network-instance/openconfig-network-instance.yang

```diff
@@ -565,7 +574,9 @@ module openconfig-network-instance {
               this list when the relevant protocol context is enabled.
               i.e., when a BGP instance is created with IPv4 and IPv6
               address families enabled, the protocol=BGP,
-              address-family=IPv4 table is created by the system.";
+              address-family=IPv4 table is created by the system.  If
+	      all BGP instances for an address family are deleted, the
+	      associated table must also be deleted.";
```

Comment:
how about `the removal of the table should not require additional or explicit configurations`?

Reply:
Done